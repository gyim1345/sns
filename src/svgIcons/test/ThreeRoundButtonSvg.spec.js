import React from 'react';
import { mount } from 'enzyme';
import ThreeRoundButtonSvg from '../ThreeRoundButtonSvg';

describe('<ThreeRoundButtonSvg />', () => {
  it('스냅샷 비교', () => {
    const wrapper = mount(<ThreeRoundButtonSvg />);
    expect(wrapper).toMatchSnapshot();
  });
})

