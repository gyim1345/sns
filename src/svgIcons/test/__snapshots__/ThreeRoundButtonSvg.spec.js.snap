// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ThreeRoundButtonSvg /> 스냅샷 비교 1`] = `
<ThreeRoundButtonSvg>
  <svg
    aria-label="옵션 더 보기"
    className="_8-yf5 "
    fill="#262626"
    height="16"
    onClick={[Function]}
    style={
      Object {
        "marginRight": "8px",
      }
    }
    viewBox="0 0 48 48"
    width="16"
  >
    <circle
      clipRule="evenodd"
      cx="8"
      cy="24"
      fillRule="evenodd"
      r="4.5"
    />
    <circle
      clipRule="evenodd"
      cx="24"
      cy="24"
      fillRule="evenodd"
      r="4.5"
    />
    <circle
      clipRule="evenodd"
      cx="40"
      cy="24"
      fillRule="evenodd"
      r="4.5"
    />
  </svg>
</ThreeRoundButtonSvg>
`;
